library/init/meta/converter/interactive.lean

```diff
@@ -30,6 +30,7 @@ open lean
 open lean.parser
 open interactive
 open interactive.types
+open tactic_result
```

Comment:
I can't quite work out why the `success` and `exception` below don't resolve to `tactic_result.success` and `tactic_result.exception`.